.github/workflows/tests.yml

```diff
@@ -140,7 +140,13 @@ jobs:
 
           - description: JavaScript component tests
             name: test-component
-            run: npx percy exec -- jest --color --selectProjects "JavaScript component tests"
+            run: npx jest --color --maxWorkers=2 --selectProjects "JavaScript component tests"
```

Comment:
Without the CPU load from the [@percy/puppeteer](https://docs.percy.io/docs/puppeteer) Chromium tabs we can now run **JavaScript component tests** with 2x workers

We use the same flag for **JavaScript behaviour tests** above